.github/workflows/cmake_nagfor.yml

```diff
@@ -55,9 +55,9 @@ jobs:
           export FFLAGS
           export FC=nagfor
           cmake -DCMAKE_BUILD_TYPE=RelWithDebInfo -DCMAKE_INSTALL_PREFIX=. -LAH -DCMAKE_C_FLAGS="" -DCMAKE_Fortran_FLAGS="${FFLAGS}" .
-          cmake --build . --target install --parallel 4
```

Comment:
why remove the parallel build option ?

Reply:
It helps with debugging the build in case something is failing, and the parallelization doesn't really speed up the test significantly.